src/lib/tabs/tab.html

```diff
@@ -1,4 +1,9 @@
 <!-- Create a template for the content of the <md-tab> so that we can grab a reference to this
     TemplateRef and use it in a Portal to render the tab content in the appropriate place in the
     tab-group. -->
-<ng-template><ng-content></ng-content></ng-template>
+<ng-template #bodyTemplate>
+  <ng-content></ng-content>
+  <ng-template *ngIf="templateBody && templateBody.template" 
```

Comment:
Why use `ngTemplateOutlet` here instead of just creating a `TemplatePortal` directly from the `TemplateRef` from `mdTabContent`?

Reply:
You need the original outlet for the projection, once projected then your able to read the template out and attach. I could replace the original outlet once projected but I'd have to attach/detach/attach. Is that what your suggesting?

Reply:
I don't really follow what you mean.

I was thinking something like
```ts
export class MdTab {
  @ContentChild(MdTabContent, {read: TemplateRef}) _explicitContent: TemplateRef<any>; 
  @ViewChild(TemplateRef) _implicitContent: TemplateRef<any>;
}
```

Then use `_explicitContent` if it is defined, otherwise use `_implicitContent` (with the same code path we currently have for `_content`).

Reply:
You are right, querying works even though its not projected. Updated per feedback.